sawtooth/cli.py

```diff
@@ -25,10 +25,12 @@
 
 from colorlog import ColoredFormatter
 
-from gossip.common import json2dict
-from sawtooth.client import SawtoothClient
+from gossip.common import json2dict, pretty_print_dict
+from sawtooth.client import SawtoothClient, LedgerWebClient
 from sawtooth.exceptions import ClientException
 from sawtooth.exceptions import InvalidTransactionError
+from ledger.transaction.endpoint_registry import EndpointRegistryTransaction
+from __builtin__ import int
```

Comment:
Is this needed ? 

```int``` is already available by default. 
normally __builtin__ is only used when wanting to override the default int object.
In Python3 __builtin__ won't be available as well.

see https://docs.python.org/2.7/reference/executionmodel.html